DEVELOPER_GUIDE.md

```diff
@@ -112,6 +145,49 @@ Typically an [EC2 G5](https://aws.amazon.com/ec2/instance-types/g5/) 2xlarge ins
 
 [Setup an EC2 Instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html)
 
+## Running Docker Images
+
+### Run the API Image as standalone container
```

Comment:
Add to top level list of links

- [Running Docker Images](#running-docker-images)
  - [Run the API Image as standalone container](#run-the-api-image-as-standalone-container)